app/facades/food_facade.rb

```diff
@@ -0,0 +1,15 @@
+class FoodFacade
+
+  def self.search_foods(query, page_num)
+    FoodService.search_foods(query, page_num)[:foods].map do |food_data|
+      if food_data[:ingredients].downcase.include?(query.downcase)
+        Food.new(food_data)
+      end
```

Comment:
This is definitely above and beyond the scope of the challenge and a very neat solution to make sure we're showing more substantial data than some of the results with a lot of blanks.